src/scss/components/carousel.scss

```diff
@@ -0,0 +1,24 @@
+.carousel {
+    overflow: hidden;
+}
+
+.slider {
+    display: flex;
+    flex-wrap: nowrap;
+}
+
+.slide {
+    width: 100%;
+	flex-shrink: 0;
```

Comment:
Please check the indentation on the different elements to be consistent. As well as the spacing between the different elements. Some have 1 space in between, other have 2